samples/ResourceOwnerPasswordFlow/DeveloperNotes/ConformanceProfiles/conformance-profiles.txt

```diff
@@ -0,0 +1,4 @@
+﻿http://openid.net/wordpress-content/uploads/2015/03/OpenID-Connect-Conformance-Profiles.pdf
```

Comment:
Yes. I would like to self-certify the web app that I am building. This is to tool to use, with which you already might be familiar. I've tested two of them. https://op.certification.openid.net:60000/